docs/DevelopmentPlan/DevelopmentPlan.md

```diff
@@ -15,67 +14,48 @@ Kyle D'Souza
   \...     |  \...              |      \... |
 
 
-[\[Put your introductory blurb here. Often the blurb is a brief roadmap
-of what is contained in the report. ---SS\]]{style="color: blue"}
+## Introductory Blurb 
+This document outlines the development plan for the project. Further information regarding the problem is located in ~/docs/ProblemStatementAndGoals/ProblemStatement.pdf.
 
-[\[Additional information on the development plan can be found in the
-[lecture
-slides](https://gitlab.cas.mcmaster.ca/courses/capstone/-/blob/main/Lectures/L02b_POCAndDevPlan/POCAndDevPlan.pdf?ref_type=heads).
----SS\]]{style="color: blue"}
+# 1 Confidential Information?
 
-# Confidential Information?
+This project will not contain extensive confidential information warranting a confidentiality agreement.
+After a team meeting with MILO, it was determined our team will inform the City to advise us of any confidential information.
+We will ensure it is not included in the source code or at the capstone expo.
 
-[\[State whether your project has confidential information from
-industry, or not. If there is confidential information, point to the
-agreement you have in place. ---SS\]]{style="color: blue"}
+# 2 IP to Protect
 
-[\[For most teams this section will just state that there is no
-confidential information to protect. ---SS\]]{style="color: blue"}
+The project does not have any IP to protect.
 
-# IP to Protect
+# 3 Copyright License
 
-[\[State whether there is IP to protect. If there is, point to the
-agreement. All students who are working on a project that requires an IP
-agreement are also required to sign the "Intellectual Property Guide
-Acknowledgement." ---SS\]]{style="color: blue"}
+What copyright license is your team adopting. Point to the license in
+your repo.
 
-# Copyright License
+# 4 Team Meeting Plan
 
-[\[What copyright license is your team adopting. Point to the license in
-your repo. ---SS\]]{style="color: blue"}
+## How often will you meet? where?
```

Comment:
you can remove the prompts if they're answered

Reply:
thanks, commited #2e3b75f which removes those prompts